packages/w/wangeditor5.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "wangEditor5",
```

```suggestion
  "name": "wangeditor5",
```
The `name` property should match the file name.